flink-table-store-flink/flink-table-store-flink-common/src/main/java/org/apache/flink/table/store/connector/sink/TableStoreSink.java

```diff
@@ -88,6 +90,10 @@ public ChangelogMode getChangelogMode(ChangelogMode requestedMode) {
                 return requestedMode;
             }
 
+            if (options.get(MERGE_ENGINE) == MergeEngine.AGGREGATE) {
```

Comment:
Introduce the until method `isRequestedMode`.

Reply:
Actually, this method `getChangelogMode(ChangelogMode requestedMode)` is doing these logical. I think we don't need extract another method?